yellowbrick/features/base.py

```diff
@@ -309,3 +312,130 @@ def _determine_target_color_type(self, y):
                 "could not determine target color type "
                 "from target='{}' to '{}'"
             ).format(self.target_type, self._target_color_type))
+
+
+##########################################################################
+## Projection Visualizers
+##########################################################################
+
+class ProjectionVisualizer(DataVisualizer):
+    
+    def __init__(self, transformer, ax=None, features=None, classes=None, color=None,
+             colormap=None, target_type="auto", proj_dim=2, alpha=0.75, **kwargs):
+        super(ProjectionVisualizer, self).__init__(ax=ax, features=features, 
+                                                     classes=classes, color=color,
+                                                     colormap=colormap, 
+                                                     target_type=target_type, **kwargs)
+        self.transformer=transformer
+        if proj_dim not in [2, 3]:
+            raise YellowbrickValueError("Projection dimensions must be either 2 or 3")
+        self.proj_dim = proj_dim
+        self.alpha = alpha
+        self._cax = None
+
+    @property
+    def cax(self):
+        """
+        The axes of the colorbar, right of the scatterplot.
+        """
+        if self._cax is None:
+            raise AttributeError(
+                "this visualizer does not have a histogram for the X axis"
```

Comment:
The attribute error message needs to be updated.

Reply:
Will do so!